src/benchcab/benchcab.py

```diff
@@ -226,6 +231,23 @@ def fluxsite_submit_job(self, config_path: str, skip: list[str]) -> None:
         logger.info("The NetCDF output for each task is written to:")
         logger.info(f"{internal.FLUXSITE_DIRS['OUTPUT']}/<task_name>_out.nc")
 
+    def codecov(self, config_path: str):
```

Comment:
```suggestion
    def gen_codecov(self, config_path: str):
```

Now that I think about it, can we rename this to `gen_codecov`? The functionality generates coverage reports but does not enable the code coverage functionality which may be confusing to users.